os/pm/pm_domain_register.c

```diff
@@ -81,7 +81,7 @@ int pm_check_domain(int domain_id)
  *
  ****************************************************************************/
 
-int pm_domain_register(char *domain)
```

Comment:
Could you please mrege pm_domain_register and pm_register

Reply:
done